src/plugins/kibana/public/dashboard/services/_saved_dashboard.js

```diff
@@ -28,7 +28,8 @@ define(function (require) {
           version: 1,
           timeRestore: false,
           timeTo: undefined,
-          timeFrom: undefined
+          timeFrom: undefined,
+          darkTheme: false
```

Comment:
Rather than putting a default value here you could use a `BoundToConfigObj` to have the default reflect the actual default. See [1] for implementation example.

[1] https://github.com/elastic/kibana/blob/a1f991f430e1f03412aba291c6ce88c2cc7f028f/src/ui/public/stringify/types/Percent.js#L10-L13